### ISSUE TITLE
requests not listed in requirements.txt

### ISSUE THREAD

**Issue by lgessler** [gucorpling/gitdox#71]
I did the following to attempt installing gitdox on my Ubuntu machine:

1. Enable Apache's CGI module
2. Move all files under gitdox's root directory to my machine's `/var/www/html/` directory
3. `sudo chmod +x *.py`
4. Navigate to `localhost/index.py`

At this point Apache gives me a 500 but I don't see anything in the error log (`/var/log/apache2/error.log`). I run `/usr/bin/python index.py` manually and find that requests isn't installed:

```
<!-- The above is a description of an error in a Python program, formatted
     for a Web browser because the 'cgitb' module was enabled.  In case you
     are not reading this in a Web browser, here is the original traceback:

Traceback (most recent call last):
  File "index.py", line 14, in &lt;module&gt;
    from paths import ether_url, get_menu
  File "/var/www/html/paths.py", line 1, in &lt;module&gt;
    import requests, os, platform
ImportError: No module named requests

-->
```

After I ran `sudo apt install python-requests`, the issue was resolved.

Based on this, I think `requests` should be listed in gitdox's `requirements.txt`, right?

edit: I wrote all this going off of gitdox's `Readme.md`, but the [documentation](https://corpling.uis.georgetown.edu/gitdox/doc.html) has slightly different instructions. `gitdox3.py` should also probably be in `requirements.txt`, and we should either sync the installation instructions in both places or have the installation instructions live in only one place.

**Comment by amir-zeldes** [comment]
Absolutely, feel free to add it!

**Comment by lgessler** [comment]
PR submitted for requirements, going to eventually rewrite Readme.md once I figure out everything that needs to be done for a clean installation

**Comment by amir-zeldes** [comment]
Fixed in 7bdc3ec97ee184b485ff032caf6d6b23a825e389
